idaes/generic_models/unit_models/column_models/column.py

```diff
@@ -0,0 +1,1144 @@
+        vapor_phase_list_ref = \
+            self.config.vapor_side.property_package.phase_list
+        liquid_phase_list_ref = \
+            self.config.liquid_side.property_package.phase_list
+
+        # Packing  parameters
+        self.eps_ref = Param(initialize=0.97,units=None,
+                             mutable=True,
+                             doc="Packing void space m3/m3")
+
+        self.packing_specific_area = Param(initialize=250,units=pyunits.m**2 / pyunits.m**3,
+                           mutable=True,
+                           doc="Packing specific surface area (m2/m3)")
+        
+        self.packing_channel_size = Param(initialize=0.1,units=pyunits.m,
+                           mutable=True,
+                           doc="Packing channel size (m)")
+        
+        self.hydraulic_diameter = Expression(expr=4 * self.eps_ref / self.packing_specific_area,
+                                 doc="Hydraulic diameter (m)")
+
+        # Add the integer indices along vapor phase length domain
+        self.zi = Param(self.vapor_phase.length_domain, mutable=True,
+                        doc='''Integer indexing parameter required for transfer
+                             across boundaries of a given volume element''')
+                             
+        # Set the integer indices along vapor phase length domain
+        for i, x in enumerate(self.vapor_phase.length_domain, 1):
+            self.zi[x] = i
+
+        # Unit Model Design Variables
+        # Geometry
+        self.diameter_column = Var(domain=Reals,
+                                   initialize=0.1,
+                                   units=pyunits.m,
+                                   doc='Column diameter')
+        
+        self.area_column = Var(domain=Reals,
+                               initialize=0.5,
+                               units=pyunits.m**2,
+                               doc='Column cross-sectional area')
+        
+        self.length_column = Var(domain=Reals,
+                                 initialize=4.9,
+                                 units=pyunits.m,
+                                 doc='Column length')
+
+        # Hydrodynamics
+        self.velocity_vap = Var(self.flowsheet().time,
+                                self.vapor_phase.length_domain,
+                                domain=NonNegativeReals,
+                                initialize=2,
+                                units=pyunits.m / pyunits.s,
+                                doc='Vapor superficial velocity')
+        
+        self.velocity_liq = Var(self.flowsheet().time,
+                                self.liquid_phase.length_domain,
+                                domain=NonNegativeReals,
+                                initialize=0.01,
+                                units=pyunits.m / pyunits.s,
+                                doc='Liquid superficial velocity')
+        
+        self.holdup_liq = Var(self.flowsheet().time,
```

Comment:
1. I think we should call this `liquid_fraction` - it is not a `holdup` itself. This also applies to the vapour holdup below.
2. We do not need both the variable for liquid fraction and the expression for vapor fraction. Just replace the vapor fraction with `1-liquid_fraction` wherever it appears.

Also, units should be specified (`pyunits.dimensionless`).